entries/jQuery.getScript.xml

```diff
@@ -1,128 +1,136 @@
+	} )
+	.fail( function( jqxhr, settings, exception ) {
+		$( "div.log" ).text( "Triggered ajaxError handler." );
+	} );
+		</code></pre>
+		<p>Prior to jQuery 1.5, the global <code>.ajaxError()</code> callback event had to be used in order to handle <code>$.getScript()</code> errors:</p>
+		<pre><code>
+$( "div.log" )
+	.ajaxError( function( e, jqxhr, settings, exception ) {
+		if ( settings.dataType === "script" ) {
+			$( this ).text( "Triggered ajaxError handler." );
+		}
+	} );
+		</code></pre>
+		<h4 id="caching-requests">Caching Responses</h4>
+		<p>By default, <code>$.getScript()</code> sets the cache setting to <code>false</code>. This appends a timestamped query parameter to the request URL to ensure that the browser downloads the script each time it is requested. 
+			You can override this feature by using a settings object, as of jQuery 1.12:</p>
+			<pre><code>
+$.getScript( {
+	url: "foo.js",
+	cache: true
+} );
+	</code></pre>
+	
+	<p>You can also override this feature by setting the cache property globally using <a href="/jquery.ajaxsetup/"><code>$.ajaxSetup()</code></a>:</p>
+	<pre><code>
+$.ajaxSetup( {
+	cache: true
+} );
+$.getScript( "foo.js" );
+	</code></pre>
+		
+	<p></p>
+	</longdesc>
+	<example>
+		<desc>Load the <a href="https://github.com/jquery/jquery-color">official jQuery Color Animation plugin</a> dynamically and bind some color animations to occur once the new functionality is loaded. Enable browser caching.</desc>
+		<code><![CDATA[
+var settings = {
+		url: "https://code.jquery.com/color/jquery.color.js",
 
-  // Allow user to set any option except for dataType, cache, and url
-  options = $.extend( options || {}, {
-    dataType: "script",
-    cache: true,
-    url: url
-  });
+		// Enable browser caching. Cache is highly recommended when loading static scripts such as plugins.
+		cache: true 
+	};
 
-  // Use $.ajax() since it is more flexible than $.getScript
-  // Return the jqXHR object so we can chain callbacks
-  return jQuery.ajax( options );
-};
-
-// Usage
-$.cachedScript( "ajax/test.js" ).done(function( script, textStatus ) {
-  console.log( textStatus );
-});
-]]></code>
-  </example>
-  <example>
-    <desc>Load the <a href="https://github.com/jquery/jquery-color">official jQuery Color Animation plugin</a> dynamically and bind some color animations to occur once the new functionality is loaded.</desc>
-    <code><![CDATA[
-var url = "https://code.jquery.com/color/jquery.color.js";
-$.getScript( url, function() {
-  $( "#go" ).click(function() {
-    $( ".block" )
-      .animate({
-        backgroundColor: "rgb(255, 180, 180)"
-      }, 1000 )
-      .delay( 500 )
-      .animate({
-        backgroundColor: "olive"
-      }, 1000 )
-      .delay( 500 )
-      .animate({
-        backgroundColor: "#00f"
-      }, 1000 );
-  });
+	// Override toString for backward compatibility with jQuery prior to 1.12 (though it will prevent browser caching)
+	settings.toString = function() {
```

Comment:
This seems pretty low-level, only plugin writes should be concerned about maintaining such support. I'd cut this mention.

Reply:
I don't think so, I actually added it BECAUSE I was searching for this for regular use.
Unless I completely misunderstood your intention

Reply:
I think the issue is that if you're using 1.x *at all* at this point, you should be using 1.12.  We have previously (years ago) discussed trying to document the differences in all versions and decided it was too confusing. As far as software projects go I don't think our docs are significantly different from other projects in that regard.